drivers/pds/lib/utils.go

```diff
@@ -1327,6 +1327,8 @@ func SetupPDSTest(ControlPlaneURL, ClusterType, AccountName, TenantName, Project
 	log.InfoD("Project Details- Name: %s, UUID: %s ", ProjectName, projectID)
 
 	ns, err = k8sCore.GetNamespace("kube-system")
+	log.Infof("kube-system ns-> %v", ns)
+	log.Infof("Error Get kube-system ns-> %v", err)
```

Comment:
do not log error here.
if the statements are added for debugging please remove them